src/main/java/taub/gameoflife/RleReader.java

```diff
@@ -0,0 +1,111 @@
+package taub.gameoflife;
+
+import java.io.*;
+
+
+public class RleReader
+{
+    File file;
+    BufferedReader reader;
+    GameOfLifeGrid grid;
+
+
+    public RleReader(File f, GameOfLifeGrid g) throws FileNotFoundException
+    {
+        file = f;
+        reader = new BufferedReader(new FileReader(file));
+        grid = g;
+    }
+
+
+    public GameOfLifeGrid getFileArr() throws IOException
+    {
+        String line = reader.readLine();
+        int height = 0;
+        while (line != null)
+        {
+            if (line.charAt(0) == 'x')
+            {
+                height = getHeight(line);
+            } else if (line.charAt(0) != '#') {
+                createPattern(line, height);
```

Comment:
Will this handle life that's multiple lines long?

Reply:
I'm really confused about this actually. Because I tried to test this out with the gosper glider gun, but the pattern seems to start rows then columns, which is the opposite of the regular glider gun.

Reply:
I added code for the gosper glider gun file but I had to take into account that it starts in a different direction than the glider file.